floodwatch/src/js/components/FilterParent.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import $ from 'jquery';
+import {GraphParent} from './GraphParent';
+import _ from 'lodash';
+import d3 from 'd3'
+
+type StateType = {
+  dataStackLayout: (data: Array<Array<StackedData>>) => Array<Array<StackedData>>
+};
+
+function initialState(): Object {
+  return {
+    dataStackLayout: d3.layout.stack()
+  }
+}
+
+type PropsType = {
+  side: string,
+  currentTopic: string,
+  updateMouseOver: (topic: string) => void,
+  data: Object
```

Comment:
Maybe in a future PR, we can actually fully type the `data` object here and in the parent components. It looks like its a `{ [key: string]: number }` but I'm not 100% sure.